### PR TITLE
Asm 3122 update csi provider to accept kubernetes auth

### PR DESCRIPTION
**Pull request by alikdolg**

### CODE REVIEW

**Review by d-goro — APPROVED** [review]

File: internal/config/config.go [hunk 1]
```diff
@@ -162,6 +167,8 @@ func parseParameters(secretStr, parametersStr string, defaultAkeylessGatewayURL
 	parameters.AkeylessAzureObjectID = params["akeylessAzureObjectID"]
 	parameters.AkeylessGCPAudience = params["akeylessGCPAudience"]
 	parameters.AkeylessUIDInitToken = params["akeylessUIDInitToken"]
+	parameters.AkeylessK8sServiceAccountToken = params["akeylessK8sServiceAccountToken"]
```
Comment (renanaAkeyless) [inline internal/config/config.go]: please delete AkeylessK8sServiceAccountToken
no use in the code

File: internal/config/config.go [hunk 2]
```diff
@@ -66,12 +69,14 @@ type Parameters struct {
...
-	AkeylessAzureObjectID string
-	AkeylessGCPAudience   string
-	AkeylessUIDInitToken  string
+	AkeylessAccessType             string
+	AkeylessAccessID               string
+	AkeylessAccessKey              string
+	AkeylessAzureObjectID          string
+	AkeylessGCPAudience            string
+	AkeylessUIDInitToken           string
+	AkeylessK8sServiceAccountToken string
```
Comment (renanaAkeyless) [inline internal/config/config.go]: delete AkeylessK8sServiceAccountToken no use

File: internal/config/config.go (hunk 1 above)
Comment (alikdolg) [reply internal/config/config.go]: fixed

File: internal/config/config.go (hunk 2 above)
Comment (alikdolg) [reply internal/config/config.go]: fixed

**Pull request merged**